cypress/support/step-definitions/tabs-steps.js

```diff
@@ -0,0 +1,19 @@
+import { tabs, tabById, tabContentById } from '../../locators/tabs';
+
+const TAB_CONTENT = 'Content for tab ';
+
+Then('Tabs component is align to {string}', (value) => {
+  tabs().should('have.css', 'text-align', value);
+});
+
+Then('Tabs component posistion is set to {string}', (value) => {
+  tabs().should('have.css', 'flex-direction', value);
+});
+
+Then('I open tab {int}', (id) => {
+  tabById(id).click();
+});
+
+Then('Tab {int} content is visible', (id) => {
+  tabContentById(id).should('be.visible').should('have.text', `${TAB_CONTENT}${id}`);
```

Comment:
second '.should' we could replace with the '.and' it will chain the first should

Reply:
yup